### PR TITLE
[SEMI-MODULAR] Adds chest-rig holstering and adds actual syndie bowmans to DS-2

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

See the title. The pseudo-sec bowmans DS-2 had are also gone, and use the syndie headset sprite that's already in the game. Admiral gets one too.

## How This Contributes To The Skyrat Roleplay Experience

haha `modular_skyrat/modules/sec_haul/sound/holsterout.ogg` go rustle

https://user-images.githubusercontent.com/86872535/137687171-a1c634f7-d441-4071-8a84-58d67e654f79.mp4

## Changelog

:cl:
add: chest-rig holstering
add: DS-2 syndie headsets
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -406,11 +406,54 @@
...
+		else
+			playsound(parent, "rustle", 50, TRUE, -5)
+
+	for(var/mob/viewing in viewers(user, null))
+		if(M == viewing)
+			to_chat(usr, "<span class='notice'>You put [I] [insert_preposition]to [parent].</span>")
+		else if(in_range(M, viewing)) //If someone is standing close enough, they can tell what it is...
+			viewing.show_message("<span class='notice'>[M] puts [I] [insert_preposition]to [parent].</span>", MSG_VISUAL)
+		else if(I && I.w_class >= 3) //Otherwise they can only see large or normal items from a distance...
+			viewing.show_message("<span class='notice'>[M] puts [I] [insert_preposition]to [parent].</span>", MSG_VISUAL)
```
Comment (GoldenAlpharex) [inline code/game/objects/items/storage/belt.dm:445]: This should be in a modular file, there is no reason this has to be included in this file.

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -406,11 +406,54 @@
...
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
-	STR.max_w_class = WEIGHT_CLASS_SMALL
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.set_holdable(list( // SKYRAT EDIT ADDITION
+		/obj/item/gun/ballistic/automatic/pistol,
+		/obj/item/gun/ballistic/revolver,
+	))
+
+// SKYRAT EDIT ADDITION END - HOLSTERING
```
Comment (GoldenAlpharex) [inline code/game/objects/items/storage/belt.dm:456]: I hope you realize you now need to make sure this doesn't happen in the tactical snack rig to ensure that people can't put guns in it, right?

File: code/game/objects/items/storage/belt.dm [hunk 3]
```diff
@@ -406,11 +406,54 @@
...
+	var/atom/A = parent
+
+	var/obj/item/gun/ballistic/automatic/pistol/gun_to_draw = locate() in real_location()
+	if(!gun_to_draw)
+		return ..()
+	A.add_fingerprint(user)
+	remove_from_storage(gun_to_draw, get_turf(user))
+	playsound(parent, 'modular_skyrat/modules/sec_haul/sound/holsterout.ogg', 50, TRUE, -5)
+	INVOKE_ASYNC(user, /mob/.proc/put_in_hands, gun_to_draw)
+	user.visible_message("<span class='warning'>[user] draws [gun_to_draw] from [parent]!</span>", "<span class='notice'>You draw [gun_to_draw] from [parent].</span>")
```
Comment (GoldenAlpharex) [inline code/game/objects/items/storage/belt.dm:427]: This also needs to use the span defines.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
This is also a balance change and I'd really like you to respect the PR template and actually try to convince us why this is a good thing for Skyrat in the "How This Contributes To The Skyrat Roleplay Experience" section.

A PR is supposed to convince us why this is a good change, if you can't do that, I don't see why I should be hitting the "merge" button.

File: code/game/objects/items/storage/belt.dm [hunk 4]
```diff
@@ -406,11 +406,54 @@
...
+	component_type = /datum/component/storage/concrete/chestrig // SKYRAT EDIT ADDITION BEGIN - HOLSTERING
+
+/datum/component/storage/concrete/chestrig/open_storage(mob/user)
+	if(!isliving(user) || !user.CanReach(parent) || user.incapacitated())
+		return FALSE
+	if(locked)
+		to_chat(user, "<span class='warning'>[parent] seems to be locked!</span>")
+		return
+
+	var/atom/A = parent
```
Comment (tf-4) [inline code/game/objects/items/storage/belt.dm:418]: https://github.com/tgstation/tgstation/pull/56872

**Comment by GoldenAlpharex** [comment]
No changes were made to the PR body in five days, giving you two more days to convince me that it's a good and well thought-out change before I close the PR.

**Comment by GoldenAlpharex** [comment]
I ended up giving you six days instead of two. Oh well.

**Pull request closed without merging**